src/methods/sep31Send/checkInfo.ts

```diff
@@ -37,10 +37,6 @@ export const checkInfo = async ({
     throw new Error("No `fields` object specified in /info");
   }
 
-  if (!asset.fields.transaction || isEmptyObject(asset.fields.transaction)) {
-    throw new Error("No transaction fields specified");
-  }
```

Comment:
All SEP-31 deployments must have the `fields` key and `transaction` subkey, but it's ok if the `transaction` object is empty.